expression/builtin_time_vec_test.go

```diff
@@ -240,8 +240,150 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 			constants: []*Constant{{Value: types.NewStringDatum("2019-12-04 00:00:00"), RetType: types.NewFieldType(mysql.TypeString)}},
 		},
 	},
-	ast.SubDate: {},
-	ast.AddDate: {},
+	ast.SubDate: {
+		{
+			retEvalType:   types.ETDatetime,
+			childrenTypes: []types.EvalType{types.ETString, types.ETString, types.ETString},
+			geners: []dataGenerator{
+				&dateStrGener{NullRation: 0.2, randGen: newDefaultRandGen()},
```

Comment:
Maybe `dateStrGener` should generate some zero date to cover the case of this issue.

Reply:
done